src/login/loginlog.cpp

```diff
@@ -123,16 +123,16 @@ bool loginlog_config_read(const char* key, const char* value) {
 bool loginlog_init(void) {
 	sql_handle = Sql_Malloc();
 
-	if( SQL_ERROR == Sql_Connect(sql_handle, log_db_username, log_db_password, log_db_hostname, log_db_port, log_db_database) )
+	if( SQL_ERROR == Sql_Connect(sql_handle, log_db_id.c_str(), log_db_pw.c_str(), log_db_ip.c_str(), log_db_port, log_db_db.c_str()) )
 	{
-        ShowError("Couldn't connect with uname='%s',passwd='%s',host='%s',port='%d',database='%s'\n",
-			log_db_username, log_db_password, log_db_hostname, log_db_port, log_db_database);
+		ShowError("Couldn't connect with uname='%s',passwd='%s',host='%s',port='%d',database='%s'\n",
+			log_db_id.c_str(), log_db_pw.c_str(), log_db_ip.c_str(), log_db_port, log_db_db.c_str());
```

Comment:
Please remove password here. We must have missed that when we removed this in other places..